cmd/livepeer_cli/wizard_broadcast.go

```diff
@@ -106,6 +106,7 @@ func (w *wizard) idListToVideoProfileList(idList string, opts map[int]string) (s
 	return pListBuf.String(), nil
 }
 
+//Deprecated.  Broadcasters are encouraged to use OBS.
 func (w *wizard) broadcast() {
```

Comment:
Why not just remove the `broadcast()` and `stream()` functions entirely? Is the code being used somewhere else?

Reply:
We could remove it completely.  Just thought I would keep it there for a little while in case we wanted to re-purpose it for something.